platforms/ide/tooling-api/src/crossVersionTest/groovy/org/gradle/integtests/tooling/r86/ProblemProgressEventCrossVersionTest.groovy

```diff
@@ -183,7 +184,8 @@ class ProblemProgressEventCrossVersionTest extends ToolingApiSpecification {
         return listener.problems.collect { (ProblemDescriptor) it }
     }
 
-    @ToolingApiVersion(">=8.6")
+    @ToolingApiVersion(">=8.7")
+    @TargetGradleVersion(">=8.7")
```

Comment:
❌ These tests are unrelated to the the intent of this PR. Why changing the version ranges?